{
    "type": "bundle",
    "id": "bundle--073b0898-858e-4814-9373-35f5ba1f98a5",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--4440c427-bc94-4951-bdfe-73f6d2fc8b2d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-12-12T01:53:41.691702Z",
            "modified": "2024-12-12T01:53:41.691702Z",
            "name": "CVE-2024-54484",
            "description": "The issue was resolved by sanitizing logging. This issue is fixed in macOS Sequoia 15.2. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-54484"
                }
            ]
        }
    ]
}